packages/devtools_app/lib/src/standalone_ui/ide_shared/property_editor/property_editor_inputs.dart

```diff
@@ -0,0 +1,223 @@
+// Copyright 2025 The Flutter Authors
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file or at https://developers.google.com/open-source/licenses/bsd.
+
+import 'package:devtools_app_shared/ui.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+
+import 'property_editor_controller.dart';
+import 'property_editor_types.dart';
+
+class BooleanInput extends StatelessWidget {
+  const BooleanInput({
+    super.key,
+    required this.property,
+    required this.controller,
+  });
+
+  final FiniteValuesProperty property;
+  final PropertyEditorController controller;
+
+  @override
+  Widget build(BuildContext context) {
+    return _DropdownInput<Object>(property: property, controller: controller);
```

Comment:
should this be `bool` instead of `Object`?

Reply:
It should be Object because we let the user set the value to either a boolean value or an expression (which is a string). I can add a comment to clarify.